package-examples/cert-manager-basic/cainjector/clusterrole-cert-manager-cainjector.yaml

```diff
@@ -1,3 +1,17 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
hmm...So these manifests are actually sourced from cert-manager release bundle, so I am not sure if we want to use our license header on these. And the original manifests didn't have any license headers.

is it possible to skip certain directories from license-checks ?

Reply:
Yeah. I've updated the PR to exclude the three packages.

Reply:
Thank you!